images/kubeadm/Dockerfile

```diff
@@ -30,7 +30,6 @@ RUN apt-get update && apt-get install -y \
 
 ENV TERRAFORM_VERSION=0.9.4 \
     KUBERNETES_PROVIDER=kubernetes-anywhere \
-    KUBECTL_VERSION=1.6.4 \
```

Comment:
no idea why this was added here but hmm
/shrug

Reply:
yeah, I mean we could keep upgrading this but... there's also a kubectl provided in the image layer underneath (kubekins-e2e), which currently tacks the stable one gcloud ships but could be switched for any other stable kubectl